modules/creating-customer-seccomp-profile.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * security/seccomp-profiles.adoc
+
+:_content-type: PROCEDURE
+[id="spo-creating-profiles_{context}"]
+= Creating seccomp profiles
+
+Use the `SeccompProfile` object to create profiles.
+
+`SeccompProfile` objects can restrict system calls (syscalls) within a container, limiting the access of your application.
+
+.Prerequisites
+
+* You have cluster admin permissions.
+* You have created a custom security context constraints (SCC). For more information, see "Additional resources".
```

Comment:
Some of the other peer reviewers are talking about the guidelines on saying something like "see, Additional resources". Will update this thread after discussion.

Reply:
After discussing, there is not an actual rule about it yet, but It it more recommended to put the title of the section thats in the Additional resources section.

Reply:
Thanks for the follow up! A quick scan of our docs shows it's referred in italics, and I like that better (for now). But we should establish some guidance on it one of these days!